docs/data/date-pickers/fields/fields.md

```diff
@@ -33,23 +23,17 @@ The field components requires an adapter imported from `@mui/x-date-pickers` or
 
 All fields to edit a range are available in a single input version and in a multi input version.
 
-:::info
-The single input version of the Time Range Field and the Date Time Range Field will soon be available.
-:::
-
 {{"demo": "DateRangeFieldExamples.js", "defaultCodeOpen": false}}
 
 ### Usage with the pickers
 
-The picker components (`DatePicker`, `DestkopDatePicker`, `MobileDatePicker`, `DateTimePicker`, ...) do not use the field components yet.
-
-New pickers using them will be released in a few months.
-
-## Form props
+The picker components (e.g. `DatePicker`) are using the field to handle their input editing.
```

Comment:
If you have other things to add here, it's very short :laughing:

Reply:
You could also remove this section, and let the following warning be in the previous section

> The single input range fields are not supported...

I assume most of the reader will notice that fields are also present in other pages

Reply:
I moved the warning to `Fields to edit a range` as proposed :+1: